docs/source/driver-framework/drivers-overview.rst

```diff
@@ -76,9 +74,9 @@ Connectivity of the platform follows the following paradigm:
 
 * Platform agents (including the Platform Driver and Actuator), subsystems, and user agents communicate with the message
   bus via a publish/subscribe system.
-* Agents can communicate "directly" to each other via JSONRPC calls - JSONRPC calls use the VOLTTRON message bus router
+* Agents can communicate "directly" to each other via JSONRPC calls. JSONRPC calls use the VOLTTRON message bus router
   to "direct" messages to an intended recipient. RPC calls from an agent specify a function for the recipient to
-  perform including input parameters, and the response to the sender should contain the value output by the specified
+  perform including input parameters; the response to the sender should contain the value output by the specified
```

Comment:
not sure I like this sentence, but can't think of how to say it better other than to put it in a glossary and put it there.

An RPC function communicates between two processes.  A caller can pass arguments to the other process.  The callee may return values.

LOL

Reply:
Yeah, now that I reread the sentence, there's a lot going on. Perhaps writing a concrete example would help? I just wrote one up and included it in the description. Let me know what you think.